src/entry/Entry.tsx

```diff
@@ -12,6 +12,9 @@ import { SequenceViewEntry } from '../model/SequenceView';
 import TissueSpeficity from '../model/TissueSpeficity';
 import Induction from '../model/Induction';
 import EntrySectionType from '../model/types/EntrySection';
+import { Keyword } from '../model/Keyword';
+// import entrySectionToKeywordCategories from '../data/keywords';
+import KeywordCategoryType from '../model/types/keywordTypes';
```

Comment:
This doesn't seem to get used in this class?

Reply:
Removed.